index.html

```diff
@@ -24,6 +24,11 @@
       <a href="#" id="logo">My logo</a>
       <nav class="nav">
         <img class="union" src="images/Union.png" alt="img" />
+        <ul class="top_navbar">
+          <li class="navbar_li">Portfolio</li>
```

Comment:
- [ ] wonderful job so far. please maybe in the course of implementing the changes you changed the background color of the mobile headline section as it was fine before. kindly make it white like in the original design . please fix that

yours             |  figma
:-------------------------:|:-------------------------:
![Screenshot (924)](https://user-images.githubusercontent.com/65298048/229269379-c48fe6c0-eced-420a-9dca-7508d9d5c5d2.png)  |  ![Screenshot (925)](https://user-images.githubusercontent.com/65298048/229269389-ea82e62e-7abb-47e3-8d37-96030dec4d36.png)